scipy/stats/__init__.py

```diff
@@ -468,6 +476,7 @@
 from ._rvs_sampling import rvs_ratio_uniforms, NumericalInverseHermite  # noqa
 from ._page_trend_test import page_trend_test
 from ._mannwhitneyu import mannwhitneyu
+from ._distn_infrastructure import fit
```

Comment:
Yeah that was something I meant to ask about. I put it there too write the code and figured we'd discuss. Also, I didn't think it would be so many lines. I'm happy with a new file, but it's been discouraged before so I don't jump into it.

Reply:
Moving to a new file will erase a lot of comment history. Let me know if your other comments have been resolved and I'll move this to `_fit`.